tests/unit/package_managers/bundler/test_parser.py

```diff
@@ -23,6 +24,15 @@
 from tests.common_utils import GIT_REF
 
 
+def relaxed_in(substring: str, messages: Iterable[str]) -> bool:
```

Comment:
This name doesn't really hint at what it's doing. Additionally, since you already went down the road, could you generalize it even more and accept a regex pattern and match that?